lib/ProductOpener/Users.pm

```diff
@@ -329,6 +330,13 @@ sub check_user_org ($user_ref, $new_org_id) {
 	return;
 }
 
+sub is_suspicious_name ($value) {
+	# email or xxx.nunsrt
+	my $email_re = qr/^[\w_.+]+(?:@[\w._+]+)?$/;
+	my $invite_re = qr/(?:click here|wants to meet you|:\/\/|\.\w{2,3}\b)/i;
```

Comment:
I'm not sure about the dot + 2 alphanumeric characters, that seems very broad. It would be good to check all existing accounts that match it, to see if there are false positives.

Maybe change \w to [a-z] to limit a bit the possibilities.

Reply:
this is because "dot + 2 alphanumeric characters" is a bit wide that I added email_re which removes more than emails, but also things where there is only one word.

Reply:
ok for [a-z]